partials/splash.php

```diff
@@ -0,0 +1,44 @@
+<div class="splash-container">
+<?php
+$images = IGV_get_option('_igv_splash_images');
+
+if ($images) {
+?>
+  <div class="splash-swiper swiper-container">
+    <div class="swiper-wrapper">
+<?php
+  foreach($images as $image) {
+    $image_id = attachment_id_from_url($image);
```

Comment:
damn sucks that CMB doesnt return IDs here like it normally does. I guess it could be a repeatable but the select all is a better UX. However we could use the Gallery Plugin which would make reordering much easier. The Gallery needs fixes from niceshirt pulled back to our forked repo